.cspell.json

```diff
@@ -38,7 +38,8 @@
     "**/log/**",
     "**/package-lock.json",
     "**/public/**",
-    "**/reports*/**"
+    "**/reports*/**",
+    "perception/bytetrack/lib/*"
```

Comment:
I think this shouldn't be included because it's a repository-specific configuration.

```suggestion
```

Reply:
Thank you for tell me that!
I remove path to bytetrack's lib from ignorePaths